src/pages/home/report/ReportActionItemSingle.tsx

```diff
@@ -220,7 +220,7 @@ function ReportActionItemSingle({
         }
         return (
             <UserDetailsTooltip
-                accountID={Number(actorAccountID ?? -1)}
```

Comment:
Using icon to keep a single source of truth in `UserDetailsTooltip` makes sense to me but I think we should still fix the `actorAccountID` value. The status is still broken (showing the status of the approver for both accounts)

<img width="520" alt="Screenshot 2024-10-21 at 9 04 58 PM" src="https://github.com/user-attachments/assets/6b2e9b86-8a2c-4fe5-be09-acb0221266dc">